src/main/java/core/basesyntax/fruittransact/FruitTransaction.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.fruittransact;
+
+public interface FruitTransaction {
+    void handle(String name, int amount);
+}

```

Comment:
transaction should be model, not inteface
it should hold type of operation, name and amount of fruit